test/integration/crcsuite/crcsuite.go

```diff
@@ -80,10 +80,11 @@ func FeatureContext(s *godog.Suite) {
 			usr, _ := user.Current()
 			CRCExecutable = filepath.Join(usr.HomeDir, "go", "bin")
 		}
+		OCExecutable := filepath.Join(CRCHome, "bin", "oc")
```

Comment:
This does not need to be exported, so `ocExecutable` should work too. However this commit makes me wonder if the clicumber changes were not too drastic, as now a clicumber test case can no longer change environment variables, which was the case before. Wondering if there could be other side effects.